router/middlewares/middlewareJson.go

```diff
@@ -8,6 +8,9 @@ import (
 	"go.uber.org/zap"
 )
 
+// A JSONHandler is a function that returns a Marshable object and/or an [github.com/ditrit/badaas/httperrors.HTTPError]
+type JSONHandler func(w http.ResponseWriter, r *http.Request) (any, httperrors.HTTPError)
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.